src/cli/self_update/test.rs

```diff
@@ -5,39 +5,34 @@ use std::sync::Mutex;
 use lazy_static::lazy_static;
 #[cfg(not(unix))]
 use winreg::{
-    enums::{RegType, HKEY_CURRENT_USER, KEY_READ, KEY_WRITE},
+    enums::{HKEY_CURRENT_USER, KEY_READ, KEY_WRITE},
     RegKey, RegValue,
 };
 
 #[cfg(not(unix))]
-use crate::utils::utils;
-
-#[cfg(not(unix))]
-pub fn get_path() -> Option<String> {
+pub fn get_path() -> Option<RegValue> {
```

Comment:
This would be a larger change, but how do you feel about changing the signature to Result<Option<RegValue>> rather than panicing ? We have other test stuff to unwind, and generally we've been heading towards Result...

Reply:
Where and how should we handle this result then ?
- In `with_saved_path` or in each test ?
- By ignoring the test or by panicking ?